CMakeLists.txt

```diff
@@ -169,9 +169,13 @@ set(TR_VCS_REVISION_FILE "${PROJECT_SOURCE_DIR}/REVISION")
 
 ## Compiler standard version
 
-set(CMAKE_C_STANDARD 11)
+if(NOT CMAKE_C_STANDARD)
+    set(CMAKE_C_STANDARD 11)
+endif()
 set(CMAKE_C_STANDARD_REQUIRED ON)
```

Comment:
I'd move `CMAKE_{C,CXX}_STANDARD_REQUIRED` definitions under matching `if(NOT CMAKE_{C,CXX}_STANDARD)` (as we do for e.g. libutp) — if we're not setting the standard, we don't have the right to enforce whatever is set externally and it's better to let it be decided externally as well.